### ISSUE TITLE
check status of Lure class

### ISSUE THREAD

**Issue by Simo-climb** [AGLDWG/phs-ont#50]
there is a lure node shape in the shapes working set. should there be a lure class in the main phs ontology. Or if deprecated from the ontology, ammend the lure node shape to the appropriate class. lets review @F3RR3T

**Comment by F3RR3T** [comment]
Yes there are some errors in the vocab. We flattened the structure so that all lures/attractants/ are now just lureTypes (we had a structure of Bait > Attractant | Lure)

**Comment by F3RR3T** [comment]
I believe you fixed this during our session.
